addons/sale/static/src/js/product/product.js

```diff
@@ -23,6 +23,7 @@ export class Product extends Component {
         exclusions: Object,
         parent_exclusions: Object,
         parent_product_tmpl_ids: { type: Array, element: Number, optional: true },
+        price_info: { type: String, optional: true },
```

Comment:
Yes, that's what I did initially, but @Feyensv suggested otherwise [1]. TBH, I have no strong opinion regarding this. Both options are fine IMO.

[1] https://github.com/odoo/enterprise/pull/56469#discussion_r1537420316